spectral_cube/base_class.py

```diff
@@ -280,6 +280,19 @@ def filled(self, fill_value=None):
                               self.unit, copy=False)
         return self.filled_data[:]
 
+    @cube_utils.slice_syntax
+    def unitless_filled_data(self, view):
+        """
+        Return a portion of the data array, with excluded mask values
+        replaced by :meth:`fill_value`.
```

Comment:
Is `fill_value` really a method...?  Maybe it is right to reference it this way?